bokeh/charts/_builder.py

```diff
@@ -76,8 +81,26 @@ class Builder(HasProps):
 
     x_range = Instance(Range)
     y_range = Instance(Range)
+    y_names = Seq(String)
+    x_names = Seq(String)
```

Comment:
I have a preference for the previous suggestion... @bryevdv just to make sure I understand your pov. In this case you propose to make all current bulders to inherit from TabularSourceBuilder, right?